src/validate.ts

```diff
@@ -5,17 +5,17 @@ const ajv = new Ajv({
   allErrors: true,
 });
 
-const syncResponseSchema = require('../intents/sync.response.schema.json');
-const queryResponseSchema = require('../intents/query.response.schema.json');
-const executeResponseSchema = require('../intents/execute.response.schema.json');
-const disconnectResponseSchema = require('../intents/disconnect.response.schema.json');
+const SYNC_RESPONSE_SCHEMA = require('../intents/sync.response.schema.json');
+const QUERY_RESPONSE_SCHEMA = require('../intents/query.response.schema.json');
+const EXECUTE_RESPONSE_SCHEMA = require('../intents/execute.response.schema.json');
+const DISCONNECT_RESPONSE_SCHEMA = require('../intents/disconnect.response.schema.json');
 
 /**
  * Helper function that uses AJV library to validate the response against the schema
  * @param apiResponse User defined api response
  * @return Returns undefined if valid is true, returns an array of error(s) if valid is false.
  */
-function responseValidation(apiResponse: object, schema: object) {
+function responseValidation(apiResponse: any, schema: any) {
```

Comment:
Is there a reason the parameters have changed to `any`? Keeping as `object` would be slightly more precise.

Reply:
Forgot to change this back, when I was debugging I kept getting an error that it wasn't of a certain type so I changed it to any to see if that's where the errors were lying. editing it back to object

Reply:
It can help to switch to the 'Files' tab after a push, and you can make it simpler by filtering up to a certain commit to verify the changes made are all intended.